lib/src/widgets/email_password_login.dart

```diff
@@ -133,101 +133,99 @@ class _EmailPasswordLoginFormState extends State<EmailPasswordLoginForm> {
-                            Spacer(flex: options.spacers.spacerAfterForm!),
-                          ],
-                          const SizedBox(height: 8),
-                          AnimatedBuilder(
-                            animation: _formValid,
-                            builder: (context, _) => options.loginButtonBuilder(
+                        ),
+                        if (widget.onForgotPassword != null) ...[
+                          Align(
+                            alignment: Alignment.topRight,
+                            child: options.forgotPasswordButtonBuilder(
                               context,
-                              _handleLogin,
-                              !_formValid.value,
                               () {
-                                _formKey.currentState?.validate();
-                              },
-                              options,
-                            ),
-                          ),
-                          if (widget.onRegister != null) ...[
-                            options.registrationButtonBuilder(
-                              context,
-                              () async {
-                                widget.onRegister?.call(
-                                  _currentEmail,
-                                  _currentPassword,
-                                );
+                                widget.onForgotPassword?.call(_currentEmail);
                               },
                               false,
                               () {},
                               options,
                             ),
-                          ],
-                          if (options.spacers.spacerAfterButton != null) ...[
-                            Spacer(flex: options.spacers.spacerAfterButton!),
-                          ],
+                          ),
                         ],
-                      ),
+                        if (options.spacers.spacerAfterForm != null) ...[
+                          Spacer(flex: options.spacers.spacerAfterForm!),
+                        ],
+                        AnimatedBuilder(
+                          animation: _formValid,
+                          builder: (context, _) => options.loginButtonBuilder(
+                            context,
+                            _handleLogin,
+                            !_formValid.value,
+                            () {
+                              _formKey.currentState?.validate();
+                            },
+                            options,
+                          ),
+                        ),
+                        if (widget.onRegister != null) ...[
+                          options.registrationButtonBuilder(
+                            context,
+                            () async {
+                              await SystemChannels.textInput.invokeMethod(
```

Comment:
could you remove this for now?
We will probably handle this in the userstory instead of the login component